### PR TITLE
Periodically send ping events on WS connections to keep them open

### PR DESCRIPTION
**Pull request by divarvel**

### CODE REVIEW

File: src/models/events.js [hunk 1]
```diff
@@ -11,13 +11,17 @@ Event.getEventsFromWS = function(url, authorization) {
...
     var ws = new WebSocket(url);
+    var pingInterval;
 
     ws.on("open", function open() {
       Logger.debug("Websocket opened successfully.")
       ws.send(JSON.stringify({
         message_type: "oauth",
         authorization: authorization
       }));
+      pingInterval = setInterval(() => {
```
Comment (BlackYoup) [inline src/models/events.js]: arrow functions are supported without `use-strict` on node 4 ?

File: src/models/log.js [hunk 2]
```diff
@@ -12,13 +12,17 @@ var Log = module.exports;
...
     var ws = new WebSocket(url);
+    var pingInterval;
 
     ws.on("open", function open() {
       Logger.debug("Websocket opened successfully.");
       ws.send(JSON.stringify({
         message_type: "oauth",
         authorization: authorization
       }));
+      pingInterval = setInterval(() => {
```
Comment (BlackYoup) [inline src/models/log.js]: Same question here ?

**Comment by divarvel** [comment]
I'll close this PR and extract the ws code into a single module (events WS
does not auto-reconnect)

**Comment by divarvel** [comment]
@BlackYoup please see https://github.com/CleverCloud/clever-tools/pull/148

**Pull request closed without merging**